parkings/importers/permit_areas.py

```diff
@@ -0,0 +1,110 @@
+import logging
+
+from django.contrib.gis.geos import MultiPolygon, Point, Polygon
+from django.db import transaction
+from lxml import etree
+from owslib.wfs import WebFeatureService
+
+from parkings.models import PermitArea
+
+logger = logging.getLogger(__name__)
+
+
+class PermitAreaImporter(object):
+
+    """
+    Imports permit area data from kartta.hel.fi.
+    """
+
+    def __init__(self, overwrite=False):
+        self.ns = {
+            'wfs': 'http://www.opengis.net/wfs/2.0',
+            'avoindata': 'https://www.hel.fi/avoindata',
+            'gml': 'http://www.opengis.net/gml/3.2',
+        }
+        self.created = 0
+
+    def import_permit_areas(self):
+        response = self._download()
+        permit_areas_dict = None
+
+        if response is not None:
+            PermitArea.objects.all().delete()
```

Comment:
Rather than deleting them all, create a list of the ids that were imported and delete everything else. This way it would stay as atomic operation and wouldn't clear anything if there's an error.